src/test/resources/features/TestAlertsAddedReview.feature

```diff
@@ -0,0 +1,149 @@
+Feature: When a prisoner with a booked visit  has high priority alerts added then visit needs review
+
+  @test_suite
+  Scenario Outline: A prisoner with a booked visit has a high priority alert (C1) added then visit needs review
```

Comment:
instead of using Scenario Outline, use Scenario. Then where you have the dynamic fields like <userName> and <password> , inline the values from the Example table below and delete the example table

Reply:
Updated